grails-app/conf/Config.groovy

```diff
@@ -576,7 +580,17 @@ openboxes.locale.supportedLocales = ['ar', 'en', 'fr', 'de', 'it', 'es', 'pt', '
 // Currency configuration
 openboxes.locale.defaultCurrencyCode = "USD"
 openboxes.locale.defaultCurrencySymbol = "\$"
-//openboxes.locale.supportedCurrencyCodes = ["USD","CFA"]
+openboxes.locale.supportedCurrencyCodes = ["USD","CAD","EUR","GBP"]
```

Comment:
For USD there is symbol included, what about other ones?

Reply:
So I don't know if the symbol is used at the moment, but I'm happy only ever displaying the symbol of the default system currency and none of the others. If we ever want to a display symbol for all currencies we should add it as a column to the UnitOfMeasure table.

Reply:
@jmiranda Will we only support these 4 currencies?

Reply:
@DominikaHatala you're right, beside what is already there i think Kelsey talked about Yen, South African Rand and one other.